stablehlo/reference/Interpreter.cpp

```diff
@@ -100,6 +100,11 @@ llvm::Expected<SmallVector<Tensor>> eval(func::FuncOp func,
       Tensor runtimeOperand = fetchOperand(negOp.getOperand());
       Tensor runtimeResult = eval(negOp, runtimeOperand);
       populateResults({runtimeResult});
+    } else if (auto multiplyOp = dyn_cast<MulOp>(op)) {
+      Tensor runtimeLhs = fetchOperand(multiplyOp.getLhs());
+      Tensor runtimeRhs = fetchOperand(multiplyOp.getRhs());
+      Tensor runtimeResult = eval(multiplyOp, runtimeLhs, runtimeRhs);
+      populateResults({runtimeResult});
```

Comment:
Similar comment about alphabetical order here. `MulOp` would go before `NegOp`. (And below in `Ops.cpp` and `Ops.h` as well).

Reply:
actually neg is above max and min so I did not see that, will move neg below